README.md

```diff
@@ -2,6 +2,160 @@
 
 Command-line interface for Jetfire/Transformations.
 
+## Usage
+
+### Authenticate
+
+To use `jetfire-cli`, the `JETFIRE_API_KEY` environment variable must be set to a valid API key for a service account which has access to Jetfire/Transformations.
+
+By default, `jetfire-cli` runs against the main CDF cluster (europe-west1-1).
+To use a different cluster, specify the `--cluster` parameter. Note that this is a global parameter, which must be specified before the subcommand.
+For example:
+
+```
+jetfire --cluster=greenfield <subcommand> [...args]
+```
+
+### Deploy transformations
+
+The primary purpose of `jetfire-cli` is to support continuous delivery, allowing you to manage transformations in a version control system.
+
+Transformations are described by YAML files, whose structure is described further below in this document.
+
+It is recommended to place these manifest files in their own directory, to avoid conflicts with other files.
+
+#### From the command line
+
+To deploy a set of transformations, use the `deploy` subcommand:
+
+```
+jetfire deploy <path>
+```
+
+The `<path>` argument should point to a directory containing YAML manifests.
+This directory is scanned recursively for `*.yml` and `*.yaml` files, so you can organize your transformations into separate subdirectories.
+
+#### From a GitHub Workflow
+
+`jetfire-cli` also provides a GitHub Action, which can be used to deploy transformations.
+To deploy a set of transformations in a GitHub workflow, add a step which references the action in your job:
+
+```yaml
+- name: Deploy transformations
+  uses: cognitedata/jetfire-cli@v1
+  with:
+    path: transformations
```

Comment:
Relative to the GitHub repository root, I suppose? Maybe we should explain that.

Reply:
Yes, absolutely!